ui/src/modal.ts

```diff
@@ -1,9 +1,8 @@
 import { derived, get, writable } from "svelte/store";
 
 type OnHide = () => void;
-const doNothing = () => {
-  /**/
```

Comment:
This was messing up my syntax highlighting.

Reply:
Which syntax highlighting do you use? Might be good to use the same

Reply:
I believe it's the Neovim built-in typescript syntax highlighting.

[`/usr/local/Cellar/neovim/0.4.4/share/nvim/runtime/syntax/typescript.vim`](https://github.com/neovim/neovim/blob/v0.4.4/runtime/syntax/typescript.vim)

<img width="553" alt="Screenshot 2021-03-09 at 10 43 50" src="https://user-images.githubusercontent.com/158411/110451831-e347a200-80c4-11eb-97e4-3c3d8c78acf1.png">